File: C/for loop/fibonaci.c
/*Display Fibonacci series up to 10 terms*/

#include <stdio.h>

int main() {
    int i;
    int a = 0, b = 1, c;
    printf("%d %d ", a, b);

    for(i = 3; i <= 10; i++) {
        c = a + b;     
        printf("%d ", c);
        a = b;         
        b = c;
    }

    return 0;
}

File: java/for loop/fibonaci.java
/*Display Fibonacci series up to 10 terms*/

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        int i;
        int a = 0, b = 1, c;
        Scanner sc = new Scanner(System.in);
        System.out.println(a+"\n"+ b);

        for(i = 3; i <= 10; i++) {
        c = a + b;     
        System.out.println(c);
        a = b;         
        b = c;
    }
    }
}